models/pod.go

```diff
@@ -33,7 +33,7 @@ type ContainerInfo struct {
 	Image string `json:"image"`
 }
 
-// Parse extracts desired information from k8s []Pod info
+// ParseDeployment extracts desired information from k8s []Pod info
```

Comment:
FYI - this is ok for me as we never choose to enforce golint / govet or whatsoever.
But just to let you know, as you may find regularly this kind of pattern from me, I'm putting the name of the function as the first word on such comments because it's a golint/govet/something rule (I can't remember which one exactly). See how it's displayed in my IDE if I don't do it:
![go doc](https://i.imgur.com/WbLyEqH.png)

So, again, no problem for this change, but be prepared to find more patterns like that :)